internal/ai/conversation.go

```diff
@@ -18,6 +19,9 @@ type Conversation struct {
 	Messages     []conversationTurn
 
 	maxTokens int64 // Maximum number of output tokens per response
```

Comment:
Let's rename this `maxOutputTokens` for clarity, now that we have two token limit variables